### PR TITLE
Add two new teacher questions to JS

### PR DESCRIPTION
**Pull request by imsickofmaps**

### CODE REVIEW

**Comment by imsickofmaps** [comment]
@smn ready for review

File: js_sandbox/lib/go-rts-zambia.js [hunk 1]
```diff
@@ -938,6 +940,31 @@ function GoRtsZambia() {
...
     ));
 
+    self.add_state(new FreeText(
+        "perf_teacher_reading_assessment",
+        "perf_teacher_reading_total",
+        "Enter the subtotal that the teacher achieved during the interview " +
+            "on Section 7.3. (Reading Assessment).",
+        function(content) {
+            // check that the value provided is actually decimal-ish.
+            return !Number.isNaN(Number(content));
```
Comment (smn) [inline js_sandbox/lib/go-rts-zambia.js]:
``` javascript
> !Number.isNaN(Number(''))
true
```

File: js_sandbox/lib/go-rts-zambia.js [hunk 2]
```diff
@@ -938,6 +940,31 @@ function GoRtsZambia() {
...
+    ));
+
+    self.add_state(new FreeText(
+        "perf_teacher_reading_total",
+        "perf_teacher_completed",
+        "According to your assessment records, how many of the pupils in the class " +
+            "that was observed have broken through/can read?",
+        function(content) {
+            // check that the value provided is actually decimal-ish.
+            return !Number.isNaN(Number(content));
```
Comment (smn) [inline js_sandbox/lib/go-rts-zambia.js]: ditto

File: js_sandbox/test/test-go-rts-zambia.js [hunk 3]
```diff
@@ -1650,6 +1650,135 @@ it("entering pupil engagement score subtotal incorrectly should ask pupil engage
...
+                perf_teacher_g2_pupils_present: '40',
+                perf_teacher_g2_pupils_registered: '50',
+                perf_teacher_classroom_environment_score: '10',
+                perf_teacher_t_l_materials: '5',
+                perf_teacher_pupils_books_number: '90',
+                perf_teacher_pupils_materials_score: '75',
+                perf_teacher_reading_lesson: '45',
+                perf_teacher_pupil_engagement_score: '22',
+                perf_teacher_attitudes_and_beliefs: '17',
+                perf_teacher_training_subtotal: '5'
```
Comment (smn) [inline js_sandbox/test/test-go-rts-zambia.js:1679]: Are all these answers needed for this state?
If not then please reduce, if yes then perhaps create a `get_answers()` helper function that can be re-used between these tests?

**Comment by smn** [comment]
:+1: in general other than the check for input being a number.

File: js_sandbox/lib/go-rts-zambia.js (hunk 1 above)
Comment (smn) [inline js_sandbox/lib/go-rts-zambia.js]: Apologies, I think I put you on this path a few reviews earlier.
/o\ javascript.

**Pull request merged**